File: 2524-largest-positive-integer-that-exists-with-its-negative/largest-positive-integer-that-exists-with-its-negative.py
class Solution:
    def findMaxK(self, nums: List[int]) -> int:
        nums.sort()
        num = nums
        i = 0

        for i in range(len(nums)):
            if nums[i] < 0 and abs(nums[i]) in num:
                return abs(nums[i])

            i += 1

        return -1

File: 2524-largest-positive-integer-that-exists-with-its-negative/largest-positive-integer-that-exists-with-its-negative.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var findMaxK = function(nums)
{
    nums.sort((a, b) => a - b);
    n = nums.length;

    for(let i = 0; i < n; i++)
    {
        if(nums[i] < 0)
        {
            for(let j = 0; j < n; j++)
            {
                if(nums[i] * -1 == nums[j])
                {
                    return nums[i] * -1;
                }
            }
        }
    }

    return -1
};